package/etc/conf.d/conflib/_common/t_templates.conf

```diff
@@ -138,7 +138,7 @@ template t_JSON_5424_SDATA {
 
 template t_splunk_hec {
     template('$(format-json
-        time=$(if ("${.netsource.sc4s_use_recv_time}" eq "yes") "$R_UNIXTIME" "$S_UNIXTIME")
+        time=$(if ("${.netsource.sc4s_use_recv_time}" eq "yes") ${R_UNIXTIME} ${S_UNIXTIME})
```

Comment:
Can we introduce a env variable , which can be enabled to change this to INT in place of STR and introduce a loop , Please check the feasibility..

Reply:
kindly mention it in PR merge doc